src/main/resources/static/css/particles/fonts/_index.scss

```diff
@@ -18,5 +18,6 @@ $_weights: ("Thin": weights.$thin, "Light": weights.$light, "Regular": weights.$
     font-style: styles.$normal;
     font-weight: $_weight-integer;
     src: url("../fonts/roboto/Roboto-" + $_weight-string + ".ttf") format("truetype");
-  }
-}
\ No newline at end of file
+    font-display: swap;
```

Comment:
das hat einen fout zur folge, was das layout komplett umwerfen kann. eher `fallback` als wert, aber lieber den browser entscheiden lassen (und damit einen foit riskieren).

Reply:
Ok das war mir nicht bewusst. Das Problem ist aktuell ohnehin, dass der Browser immer über das Font entschiedet (s. #28 ).

Reply:
`fallback` wäre denke ich ein guter Kompromiss. Damit bekommt auch über eine langsamere Verbindung nicht einen ewig langen FOIT. Für schnelle Verbindungen macht dass zum default Fall im Grunde auch keinen Unterschied.

Reply:
Dann müssen wir sowohl für die gewünschte, als auch für die Fallback-Font layouten. Können wir ja mal ausprobieren. :)

Reply:
Ich denke man kann das Problem minimieren, indem man `Arial` als `fallback`angibt. 
Aktuell müsste das fallback `sans-serif`, was schon einen starken Unterschied macht. 

https://meowni.ca/font-style-matcher/ 
Hier kann man die Fonts gegeneinander matchen und sich unten den Unterschied im render anschauen.

Reply:
Schön! Dann wäre ich auch mit `swap` einverstanden, sofern dann wirklich `Arial` und `Roboto` ähnlich aussehen.

Ich würde dann später `Roboto` in den Branch hier pushen.

Reply:
Alles klar, dann pack ich `Arial` als fallback rein und lass das `swap` erstmal drin.

Reply:
BTW: Sowohl Font-Family als auch Fallback werden gar nicht verwendet. Keine CSS-Angabe verweist darauf.

Ich kann es aber mal im `body` oder so setzen.

Reply:
Habe es gerade mal ausprobiert... Mit swap wirkt es unschön, da man das austauschen wirklich sieht. Ich würde vielleicht doch `fallback` oder den Browser-Standard verwenden.

Reply:
:+1: